src/Core/ServiceWrapper/winsw.csproj

```diff
@@ -1,8 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please revert BOM changes.

Reply:
what are the BOM changes? I don't know that :(

Reply:
Simply revert this line.